cvat/apps/documentation/user_guide.md

```diff
@@ -1,3 +1,7 @@
+<!--lint disable maximum-line-length-->
+<!--lint disable list-item-spacing-->
+<!--lint disable list-item-indent-->
+
```

Comment:
@zhiltsov-max , please revert these changes and ask @TOsmanov to fix trailing spaces.

Reply:
@TOsmanov, please, remove trailing spaces from the document.

Reply:
@zhiltsov-max , I remove trailing spaces in this commit https://github.com/opencv/cvat/pull/1508/commits/eaa16551bd8a35f66cd2c7d4dae9a0690b8cb67a